tutorials/R/Geospatial-skills/intro-vector-r/01-shapefile-attributes/01-shapefile-attributes.Rmd

```diff
@@ -388,7 +388,7 @@ Note in the above example we have:
 Let's give this a try.
 
 
-``` {r palette-and-plot}
+``` {r palette-and-plot, fig.cap="Roads and trails at NEON Harvard Forest Field Site with color varied by attribute factor level.}
```

Comment:
@mayastahl I think this needs a closing quote "

Reply:
Fixed it! Thanks!